src/js/form-builder.js

```diff
@@ -927,10 +929,15 @@
           id: lastID + '-edit',
           className: 'toggle-form btn icon-pencil',
           title: opts.messages.hide
+        }),
+        copyBtn = utils.markup('a', opts.messages.copyButton, {
+          id: lastID + '-copy',
+          className: 'copy-button btn icon-copy',
+          title: opts.messages.copyButtonTooltip
         });
 
       var liContents = utils.markup(
-        'div', [toggleBtn, delBtn], { className: 'field-actions' }
+        'div', [toggleBtn, copyBtn, delBtn], { className: 'field-actions' }

```

Comment:
In that case, let's leave it at `[toggleBtn, copyBtn, delBtn]` keeping the edit button on the left -- closest to the field attributes that need to be edited.